videointelligence/google/cloud/videointelligence_v1beta1/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from google.cloud.gapic.videointelligence.v1beta1.video_intelligence_service_client import VideoIntelligenceServiceClient
+from google.cloud.gapic.videointelligence.v1beta1 import enums
+
```

Comment:
Actually, it was a mistake, but it might be a good one? I went to remove it and it sort of obscured the difference between `videointelligence.v1beta1` and `videointelligence_v1beta1`. Leaving as-is for now but happy to change.